SPECS/el9/gdal.spec

```diff
@@ -267,6 +267,9 @@ export PYTEST="pytest -vv -p no:sugar --color=no"
 #  * ogr/ogr_pg: trying to use PostGIS when it's not supposed to
 #  * pyscripts/test_gdal2tiles.py: regressions with test file locations on EL.
 $PYTEST \
+--deselect ogr/ogr_fgdb.py::test_ogr_fgdb_19 \
```

Comment:
I thought issues were fixed with 3.6.0 as they passed fine in a local build.  Then I remembered these specifically can fail randomly, even if they pass sometimes.